### DISCUSSION TITLE
autocomplete nope

### DISCUSSION THREAD

**Discussion by japafrite** [spatie/laravel-honeypot#119] [Q&A]
Hello,
In resources/views/honeypotFormFields.blade.php there is autocomplete="nope".
Is there a special reason ? 
Because nope is not a valid value. It should be "off".

**Comment by freekmurze** [comment]
Thanks for reporting, this has been fixed!

**Reply by yannisc** [reply]
The problem is stil there on the latest version. Nope instead of off.
